impress-org/givewp-next-gen
# Feature: add test mode notice

## Description



This PR adds a test mode notice to the Payment Gateways field. This is displayed only when Give is under testing mode.

## Affects



The Payment Gateways field.

## Visuals



![image](https://github.com/impress-org/givewp-next-gen/assets/16308864/f9974a4c-59fb-43be-b174-a300e670787a)


## Testing Instructions



Just set Give mode to "test" and check if the notice is displayed when the form renders.

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205043276379238
